roadmap/implementers-guide/src/types/overseer-protocol.md

```diff
@@ -22,6 +22,17 @@ All subsystems have their own message types; all of them need to be able to list
 
 Either way, there will be some top-level type encapsulating messages from the overseer to each subsystem.
 
+## Active Leaves Update
+
+Indicates a change in active leaves. Activated leaves should have jobs, whereas deactivated leaves should lead to winding-down of work based on those leaves.
+
+```rust
+struct ActiveLeavesUpdate {
+	activated: [Hash], // in practice, these should probably be a SmallVec
```

Comment:
are these deltas or _all_ active leaves?

Reply:
Given the context (SmallVec comment, grammatical form) I'd expect these to be deltas, but agree it would be very good to make that explicit.

Reply:
`activated` implies change, `active` would be the entire set.